website/blog/2023-02-01-technical-preview/index.mdx

```diff
@@ -0,0 +1,113 @@
+---
+slug: technical-preview
+title: v0.0.1 - Technical Preview
+authors: [ pmig, lw ]
+published: false
+tags: [ glasskube, release ]
+---
+
+import Install from '../../src/partials/_install.mdx';
+
+Glasskube is open-source on GitHub - Support us by leaving a star: [⭐ `glasskube/glasskube` ⭐](https://github.com/glasskube/glasskube/)
+
+
+*The aim of this post is to share our technical preview of how a cloud native package manager could work and what challenges need to be solved.*
+
+
+## The missing package manager for Kubernetes - Introducing Glasskube
+
+Package Management on Kubernetes is one of the most pressing issues in the Cloud Native community.
+A concept which is widely known from other ecosystems like desktop and mobile computing has yet not been realized for cloud computing.
+On Android and iOS for example millions of developers publish their packages via the Play store or App Store to its users.
+The package manager also makes sure all users receive the latest version published by the developer and the developer receives
+crash reports and user feedback as a return to improve their applications,
+but as a cloud native developer there is no package manager you can rely on - yet.
+
+Our first release (v0.0.1) already features a working prototype that can install basic packages, but a lot of challenges still need to be solved.
+
+### A cloud native architecture
+
+Glasskube is designed as a cloud native application itself.
+Featuring an easy to install client that comes with a graphical user interface and autocompletion for your favorite shell.
+
+At the heart of the Glasskube package ecosystems lies our central package registry which holds the package manifests.
```

Comment:
At this point I would add that support for 3rd party registries and using multiple registries in a cluster is planned